### PR TITLE
Remove semantic-ui, host Lato locally and fix some style issues

### LINKED ISSUE #368: Héberger la police Lato en local
https://www.fontsquirrel.com/fonts/lato

Pour :

* plus de dépendance à Google
* davantage de latitude sur la stratégie de chargement (perfs)
* une dépendance à semanticui de moins

Contre :

* plus lent si on s'y prend mal :)

### PR DESCRIPTION
**Pull request by tusbar**
Fix #368

### CODE REVIEW

**Comment by tusbar** [comment]
screen shot 2017-12-10 at 4 15 54 am

On heroku.

File: util/font-loader.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  })
+  const latoItalic = new FontFaceObserver('Lato', {
+    style: 'italic'
+  })
+  const latoBoldItalic = new FontFaceObserver('Lato', {
+    weight: 700,
+    style: 'italic'
+  })
+
+  await Promise.all([
```
Comment (davidbgk) [inline util/font-loader.js]: What I used to do is more something like that:

```
(function () {

  // Optimization for Repeat Views
  if (sessionStorage.criticalFoftDataUriFontsLoaded) {
      document.documentElement.classList.add('fonts-critical', 'fonts-loaded')
      return
  }
  const fontASubset = new FontFaceObserver('LatoSubset')
  fontASubset.load().then(_ => {
    document.documentElement.classList.add('fonts-critical')

    const fontA = new FontFaceObserver('Lato')
    const fontB = new FontFaceObserver('LatoBold', {weight: 700})
    const fontC = new FontFaceObserver('LatoItalic', {style: 'italic'})
    const fontD = new FontFaceObserver('LatoBoldItalic', {weight: 700, style: 'italic'})
    Promise.all([fontA.load(), fontB.load(), fontC.load(), fontD.load()]).then(_ => {
      document.documentElement.classList.add('fonts-loaded')

      // Optimization for Repeat Views
      sessionStorage.criticalFoftDataUriFontsLoaded = true
    })
  })
})()
```

This way you start to render the most critical font first, in our (French) case you can just remove the LatoSubset and use Lato instead. Tell me if it's not clear we can setup a 10min visio about that.

Comment (tusbar) [reply util/font-loader.js]: Except for the different font names (what’s the point?), how is this different from what I did?

And yeah, an ASCII subset is not necessary.

Comment (davidbgk) [reply util/font-loader.js]: The proposed version is a two-steps process.

**Review by jdesboeufs — APPROVED** [review]
OK for me :)
Thx @davidbgk!

File: pages/_document.js [hunk 2]
```diff
@@ -12,9 +12,48 @@ export default class MyDocument extends Document {
...
+          <link rel='mask-icon' href='/static/favicons/safari-pinned-tab.svg' color='#5bbad5' />
+
+          <link rel='preload' href='/static/fonts/lato-regular-webfont.woff2' as='font' type='font/woff2' crossOrigin='' />
+        </Head>
         <body>
           <Main />
+          <script dangerouslySetInnerHTML={{
+            __html: `
+              (function () {
+                if ('fonts' in document) {
```
Comment (davidbgk) [inline pages/_document.js:32]: Beware that it excludes [all IEs so far](https://caniuse.com/#feat=font-loading), that's why I still recommend [fontfaceobserver](https://github.com/bramstein/fontfaceobserver) but YMMV.

**Comment by davidbgk** [comment]
@tusbar out of curiosity, can you retry the [previous performance check](https://github.com/etalab/geo.data.gouv.fr/pull/569#issuecomment-350522082) with the latest implementation to see if it changed something?

File: pages/_document.js (hunk 2 above)
Comment (tusbar) [reply pages/_document.js:32]: I know, but fontfaceobserver is too heavy. IE will just have to deal with FOIT in the first render.

**Comment by tusbar** [comment]
It seems a little slower, but probably unrelated.

screen shot 2017-12-11 at 3 44 50 pm

I don’t see how it’d change much.

**Comment by davidbgk** [comment]
It's hard to see when fonts are actually loaded on compressed thumbnails 😅

I'll take a look once pushed to production, thanks for the check!

**Review by davidbgk — APPROVED** [review]

**Pull request merged**